src/components/widgets/StyledComponents.tsx

```diff
@@ -163,3 +164,39 @@ export const FormControlLabelHidden = withStyles(theme => ({
     },
   },
 }))(FormControlLabel)
+
+export const AntSwitch = withStyles(theme =>
```

Comment:
since we will be using the same type of switch everywhere -- let's just override it in themes.ts
```javascript
   MuiSwitch: {
      root: {
        width: 32,
        height: 18,
        padding: 0,
        display: 'flex',
      },
      switchBase: {
        padding: 2,
        color: '#fff',
        '& + $track': {
          opacity: 1,
          backgroundColor: '#ddd'
        },
        '&$checked': {
          transform: 'translateX(14px)',
          color: '#fff',
          '& + $track': {
            opacity: 1,
            backgroundColor: '#7FC7F0',
            borderColor: '#7FC7F0',
          },
        },
      },
      thumb: {
        width: 14,
        height: 14,
      },
      track: {
        border: `1px solid #ccc`,
        borderRadius: 16 / 2,
        opacity: 1,
        backgroundColor: '#fff',
      },
      colorPrimary: {
        '&.Mui-checked': {
          color: '#fff',

          '& + .MuiSwitch-track': {
            backgroundColor: '#7FC7F0',
          },
        },
      },
      colorSecondary: {
        '&.Mui-checked': {
          color: '#fff',

          '& + .MuiSwitch-track': {
            backgroundColor: '#E7BDBD',
          },
        },
      },
    }
`

Reply:
The 'primary' would have blue background and 'secondary' would be pink